frontend/src/components/rota/rota.module.scss

```diff
@@ -1,5 +1,16 @@
 .rota {
   grid-area: main;
-  width: 100%;
-  height: 100%;
+  display: grid;
+  height: 100vh;
+  grid-template:
+    'header' min-content
+    'calendar' 1fr;
+  flex-direction: column;
```

Comment:
is this relevant for `grid` display?
thought it's just for `flex`

Reply:
Ah, I was playing around with different layouts and forgot to remove this. Will fix.